test/test_helper.rb

```diff
@@ -1,13 +1,3 @@
-ENV['RAILS_ENV'] = 'test'
-
-require 'simplecov'
```

Comment:
No more code coverage?  I guess we weren't really using it anyways.

Reply:
ok ok - I was simplifying things while getting them all to run, I'll add it back!